File: atcoder.jp/Python (3.8.2)/abc309/abc309_a.py
A, B = map(int, input().split())

ay, ax = divmod(A - 1, 3)
by, bx = divmod(B - 1, 3)

if abs(ax - bx) <= 1 and ay == by:
    print("Yes")
else:
    print("No")

File: atcoder.jp/Rust (1.42.0)/abc309/abc309_a.rs
use num_integer::div_mod_floor;
use proconio::{fastout, input};

#[fastout]
fn main() {
    input! {
        a: isize,
        b: isize,
    }

    let (ya, xa) = div_mod_floor(a - 1, 3);
    let (yb, xb) = div_mod_floor(b - 1, 3);

    if (xa - xb).abs() == 1 && ya == yb {
        println!("Yes")
    } else {
        println!("No")
    }
}
